src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -900,4 +900,16 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
   <data name="Accessibility_Packages" xml:space="preserve">
     <value>Packages</value>
   </data>
+  <data name="Status_Canceled" xml:space="preserve">
+    <value>Package results loading cancelled</value>
+  </data>
+  <data name="Status_ErrorOccurred" xml:space="preserve">
+    <value>Error occurred when loading package results</value>
+  </data>
+  <data name="Status_NoMoreItems" xml:space="preserve">
+    <value>Last package results page loaded</value>
+  </data>
+  <data name="Status_Ready" xml:space="preserve">
+    <value>Package results page loaded</value>
```

Comment:
Same here, packages page loaded maybe?

Reply:
What's the difference here compared to NoMoreItems? I don't think "page loaded" may be clear to the user.

Reply:
NoMoreItems trigges when you are at the last packages page in a paginated search

Reply:
I think I'm good with this string. Won't resolve since I jumped on @nkolev92 's thread

Reply:
Yeah I'm fine with whatever you guys decided. Resolving.